Automattic/jetpack
# VideoPress: fix replace VideoPress video

This PR fixes (again) the issue when the app tries to replace the video by selecting an item from the media gallery.


Fixes #

## Proposed changes:

* VideoPress: fix the issue when trying to replace the video from the Media Gallery

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Create/edit a video block
* Ensure the block already has a video
* Try to replace the video by selecting an item from the Media Gallery
<img width="678" alt="image" src="https://user-images.githubusercontent.com/77539/213297118-6685732a-2ea9-40fa-ba12-b01a49fd34d0.png">
* Confirm the video is replaced once the item is selected
* Confirm replacement by uploading a new file works as expected
